src/main/java/com/checkmarx/sdk/service/scanner/ScaScanner.java

```diff
@@ -199,4 +198,7 @@ private void validateNotNull(SCAResults scaResults) {
         }
     }
 
+    protected void setRemoteBranch(ScanParams scanParams, RemoteRepositoryInfo remoteRepoInfo) {
+       //branches are not supported for SCA
```

Comment:
Here SCA will scan a default branch even if the user specified a custom branch in scanParams.getBranch(). This can lead to a lot of confusion. I guess we should throw an error in this case.